kusto/ingest/streaming.go

```diff
@@ -32,7 +32,7 @@ var FileIsBlobErr = errors.ES(errors.OpIngestStream, errors.KClientArgs, "blobst
 // More information can be found here:
 // https://docs.microsoft.com/en-us/azure/kusto/management/create-ingestion-mapping-command
 func NewStreaming(client QueryClient, db, table string) (*Streaming, error) {
-	streamConn, err := conn.New(client.Endpoint(), client.Auth(), client.HttpClient())
+	streamConn, err := conn.New(client.Endpoint(), client.Auth(), client.HttpClient(), "", "", "")
```

Comment:
Is not having CRPs here mapped as a parity gap, or is it the same in all SDKs?

Reply:
Also, should we not take this from 'client'?

Reply:
No SDK lets you send CRP to streaming ingestion

Reply:
In fact, the endpoint doesn't expect it either..